Player

rakshaankhan opened the pull request:
Player Crouch and Player Sprint added, with player jump already present, in the test scene for initial movement controls. Will be working further to fix the issue with the Move Script.

rakshaankhan committed:
Player Crouch and Sprint

These are Scripts and components added to the Capsule (Player) for those specified controls in the Test scene. I will be working on correcting the initial movement to make it functional. Just pushing it here for safekeep and push my changes in.

rakshaankhan committed:
Merge branch 'player' of https://github.com/eldarb/Sort-it-out into player

kdeng2001 commented:
I would suggest removing or at least commenting out line 53 of the PlayerSprint script, where you set the rotation of the player. Once this is done, we can merge with main. Everything else looks good!

rakshaankhan committed:
PlayerSprint.cs updated

I fixed line 53 and got rid of the unnecessary line to still implement the script.
